compileopts/target.go

```diff
@@ -271,3 +272,17 @@ func defaultTarget(goos, goarch, triple string) (*TargetSpec, error) {
 	}
 	return &spec, nil
 }
+
+// LookupGDB looks up a gdb executable.
+func (spec *TargetSpec) LookupGDB() (string, error) {
+	if len(spec.GDB) == 0 {
+		return "", errors.New("gdb not configured in the target specification")
+	}
+	for _, d := range spec.GDB {
+		_, err := exec.LookPath(d)
+		if err == nil {
+			return d, nil
+		}
+	}
+	return "", errors.New("no gdb found configured in the target specification")
```

Comment:
Can you maybe include the command names that were checked in the error message? That makes it easier to know how to fix the issue based on the error message.

Reply:
Added information to error message.

```
error: no gdb found configured in the target specification (gdb-multiarch, arm-none-eabi-gdb)
```